### PR TITLE
Add timeout to infrastructure pipeline jobs

### LINKED ISSUE #974: Add timeout blocks to JenkinsFiles
Sometimes builds hang for various reasons. We should add timeout blocks to either an entire build or per stage so that things aren't running longer than they should and require manual cancelling.

### PR DESCRIPTION
**Pull request by vsebe**
Set timeout to 6 hours for copyright and lineEndings check 
and to 3 hours for the other infrastructure jobs.

Fixes #974 

[ci skip]

Signed-off-by: Violeta Sebe <vsebe@ca.ibm.com>

### CODE REVIEW

**Comment by vsebe** [comment]
@AdamBrousseau please review, thank you!

**Comment by vsebe** [comment]
@pshipton please review, thank you!

**Comment by AdamBrousseau** [comment]
Should add `fixes #974` to your commit

**Comment by vsebe** [comment]
Comment updated.

File: buildenv/jenkins/jobs/infrastructure/copyrightCheck [hunk 1]
```diff
@@ -25,62 +25,64 @@ String SRC_REPO = 'https://github.com/${ghprbGhRepository}.git'
...
-                    echo "${HASHES}"
-                    sh 'exit 1'
-                } else {
-                    echo "All modified files appear to have correct copyrights"
                 }
             }
         }
     }
-}
+}
```
Comment (pshipton) [inline buildenv/jenkins/jobs/infrastructure/copyrightCheck]: Please end the file with a new line

Comment (vsebe) [reply buildenv/jenkins/jobs/infrastructure/copyrightCheck]: Added new line, ready for review.

**Pull request merged**